### PR TITLE
Install WIFI packages during system installation bug#1002700

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: src/include/network/lan/wizards.rb [hunk 1]
```diff
@@ -55,11 +55,12 @@ def LanSequence
       aliases = {
         "read"     => [-> { ReadDialog() }, true],
         "main"     => -> { MainSequence("") },
-        "packages" => [-> { PackagesInstall(Lan.Packages) }, true],
         "write"    => [-> { WriteDialog() }, true]
```
Comment (mvidner) [inline src/include/network/lan/wizards.rb]: This change makes RuboCop fail with "Align the elements of a hash literal if they span more than one line."
W!T!F!

File: package/yast2-network.spec [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-network
-Version:        3.1.171
+Version:        3.2.1
```
Comment (mchf) [inline package/yast2-network.spec]: different version number than in spec

Comment (ancorgs) [inline package/yast2-network.spec]: I had to "correct history" (like dictators do) because the previous patch didn't jump to 3.2.0 as expected.

Comment (mchf) [inline package/yast2-network.spec]: ah sorry ... for second view I can see ...

**Pull request merged**